epochStart/bootstrap/syncEpochStartMeta_test.go

```diff
@@ -0,0 +1,115 @@
+package bootstrap
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/data/block"
+	"github.com/ElrondNetwork/elrond-go/epochStart/mock"
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/ElrondNetwork/elrond-go/process/economics"
+	"github.com/stretchr/testify/require"
+)
+
+func TestNewEpochStartMetaSyncer_ShouldWork(t *testing.T) {
+	t.Parallel()
+
+	args := getEpochStartSyncerArgs()
+	ess, err := NewEpochStartMetaSyncer(args)
+	require.NoError(t, err)
+	require.False(t, check.IfNil(ess))
+}
+
+func TestEpochStartMetaSyncer_SyncEpochStartMeta_ErrRegisterMessengerProcessorShouldErr(t *testing.T) {
```

Comment:
Maybe remove the second _ in the name in order to comply with the rest of the project (all function names)

Reply:
In some situations it is better to use another `_` as the name would be more readable and also this way you can separate the method's name and the test "description". But renamed

Reply:
I know, we have decided to have a single _ in the name 1 1/2 years ago.